### PR TITLE
M0127CC-611 Github article/tutorial

### PR DESCRIPTION
**Pull request by fpinel-microej**

### CODE REVIEW

**Comment by jerome-leroux** [comment]
Can you check the formatting of the references to the different menus?
It is not consistent with the other pages (see https://github.com/MicroEJ/docs/blob/master/PlatformDeveloperGuide/platformCreation.rst for example).

**Comment by jerome-leroux** [comment]
It would be better to start from the project import menu and clone the repository from this menu instead of doing first the clone and then the import.

File: Tutorials/tutorialGithub.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+MicroEJ and Github
+==================
```
Comment (stephanemainchain) [inline Tutorials/tutorialGithub.rst:2]: Change title to :
Importing Github projects into MicroEJ SDK

File: Tutorials/tutorialGithub.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+You may know that MicroEJ has an `account on Github <https://github.com/MicroEJ>`_. There, we share examples (like `examples for MWT <https://github.com/MicroEJ/ExampleJava-MWT>`_) libraries, tools and demos. In this article, we will explain how you can connect repositories from Gitub inside MicroEJ.
```
Comment (stephanemainchain) [inline Tutorials/tutorialGithub.rst:4]: typo Gitub = Github

File: Tutorials/tutorialGithub.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Both the MicroEJ Studio and MicroEJ SDK come with the EGit plugin built-in, so you can easily clone an existing repository. Here, we use the MWT Examples repository. From the Github page, you can copy the HTTP address of this repository from the dedicated field in the right menu (highlighted in red):
```
Comment (stephanemainchain) [inline Tutorials/tutorialGithub.rst:6]: remove "the"
Both MicroEJ Studio and SDK,

File: Tutorials/tutorialGithub.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+.. figure:: images/0-address-to-clone.png
+   :alt: Address to clone
+   :align: center
+
+For this repository, the address is `<https://github.com/MicroEJ/ExampleJava-MWT.git>`_. In MicroEJ, open the Git perspective and select “Clone a Git repository” in the “Git Repositories” view. If have already copied the HTTP address, the fields are filed automatically:
```
Comment (stephanemainchain) [inline Tutorials/tutorialGithub.rst:12]: In MicroEJ SDK, open...

**Pull request merged**